platforms/chibios/drivers/spi_master.c

```diff
@@ -38,9 +38,15 @@ __attribute__((weak)) void spi_init(void) {
 
         chThdSleepMilliseconds(10);
 #if defined(USE_GPIOV1)
+#    if defined(HT32)
```

Comment:
This should be reverted, and instead something like the following should be added to `tmk_core/common/chibios/chibios_config.h`
```c
#if defined(HT32)
#    define PAL_MODE_ALTERNATE PAL_HT32_MODE_AF
#    define PAL_OUTPUT_SPEED_HIGHEST 0
#endif
```

Reply:
fixed